### PR TITLE
Add input renderer 

### PR DESCRIPTION
**Pull request by Deivis**
Add renderer prop and tests to input component

### CODE REVIEW

File: src/Input/Input.js [hunk 1]
```diff
@@ -49,6 +51,14 @@ const validateMultiline = ({
   ) {
     throw new Error('Adding a mask to a multiline component is not possible, it was rendered without the mask.')
   }
+
+  if (
+    propName === 'multiline' &&
+    multiline &&
+    !isEmpty(renderer)
+  ) {
+    throw new Error('Adding renderer to a multiline component is not possible, it was rendered as a multiline input.')
```
Comment (LeoNero) [inline src/Input/Input.js]: I think this message should ent at `not possible, `. Also, we should write a test for this case!

File: src/Input/Input.js [hunk 2]
```diff
@@ -94,6 +104,49 @@ class Input extends React.PureComponent {
...
+      disabled,
+      mask,
+      multiline,
+      onChange,
+      renderer,
+      type,
+    } = this.props
+
+    const inputType = (type === 'password' && this.state.showPassword)
+      || multiline
```
Comment (LeoNero) [inline src/Input/Input.js]: Place it in the above line

**Comment by LeoNero** [comment]
Also, add some storybook examples :D

File: src/Input/Input.js (hunk 2 above)
Comment (rsmelo) [reply src/Input/Input.js]: passo de 80 colunas se botar na linha de cima =(

**Pull request merged**